### PR TITLE
Add lunar

### PR DESCRIPTION
**Pull request by mikaelarguedas**
Add lunar to buildfarm config + update all jobs priorities

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -41,6 +41,21 @@ distributions:
...
+      released-packages-without-doc-job: lunar/doc-released-build.yaml
+    notification_emails:
+    - ros-buildfarm-kinetic@googlegroups.com
+    - mikael+buildfarm@osrfoundation.org
+    release_builds:
+      uxhf: lunar/release-armhf-build.yaml
+      uxv8: lunar/release-arm64-build.yaml
+      ds: lunar/release-stretch-build.yaml
+      dsv8: lunar/release-stretch-arm64-build.yaml
+      default: lunar/release-build.yaml
```
Comment (dirk-thomas) [inline index.yaml]: Please keep the keys in alphabetical order.

Kinetic should probably be fixed too.

File: lunar/release-stretch-arm64-build.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - http://repositories.ros.org/ubuntu/building
+target_repository: http://repositories.ros.org/ubuntu/building
+targets:
+  debian:
+    jessie:
+      arm64:
+type: release-build
+upload_credential_id: 1e7d4696-7fd4-4bc6-8c87-ebc7b6ce16e5
+version: 2
+
```
Comment (dirk-thomas) [inline lunar/release-stretch-arm64-build.yaml]: Some build files have an empty line at the end - some don't. They should contain a newline after the `version` line but not an empty line (aka two newlines).

**Review by dirk-thomas — COMMENTED** [review]
LGTM beside the minor comments.

Can be merged after ros-infrastructure/rep#123 has been accepted.

File: lunar/release-stretch-arm64-build.yaml (hunk 2 above)
Comment (mikaelarguedas) [reply lunar/release-stretch-arm64-build.yaml]: fixed in https://github.com/ros-infrastructure/ros_buildfarm_config/pull/69/commits/a98b2fcdbf86014ff4addcac0408d37305cbe389

File: index.yaml (hunk 1 above)
Comment (mikaelarguedas) [reply index.yaml]: fixed in https://github.com/ros-infrastructure/ros_buildfarm_config/pull/69/commits/1ae376fea92e600789e4c293297db3be26a802ba

**Comment by mikaelarguedas** [comment]
Updated targeted platfroms in https://github.com/ros-infrastructure/ros_buildfarm_config/pull/69/commits/1c8965a43f2f16e8ee7db10c395d6b1a69d67c53 based on https://github.com/ros-infrastructure/rep/pull/123

File: index.yaml [hunk 3]
```diff
@@ -34,13 +34,28 @@ distributions:
...
+      uxhf: kinetic/release-armhf-build.yaml
+      uxv8: kinetic/release-xenial-arm64-build.yaml
     source_builds:
       default: kinetic/source-build.yaml
+  lunar:
+    doc_builds:
+      default: lunar/doc-build.yaml
+      released-packages-without-doc-job: lunar/doc-released-build.yaml
+    notification_emails:
+    - ros-buildfarm-kinetic@googlegroups.com
```
Comment (dirk-thomas) [inline index.yaml:49]: This references the Kinetic mailing list. Please fix.

So do all the new build files.

**Comment by dirk-thomas** [comment]
This PR was not approved yet. I would be good to wait for that before merging in the future.

**Comment by dirk-thomas** [comment]
This PR was not approved yet. I would be good to wait for that before merging in the future.

Nvm: it actually was approved before. It looks like GitHub removed the approval icon. I don't know why though...

File: lunar/release-armhf-build.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    qYE=
+    =Vgio
+    -----END PGP PUBLIC KEY BLOCK-----
+  urls:
+  - http://repositories.ros.org/ubuntu/building
+target_repository: http://repositories.ros.org/ubuntu/building
+targets:
+  ubuntu:
+    xenial:
+      arm64:
```
Comment (dirk-thomas) [inline lunar/release-armhf-build.yaml:56]: This should be `armhf`.

File: index.yaml [hunk 5]
```diff
@@ -34,13 +34,28 @@ distributions:
...
+    - mikael+buildfarm@osrfoundation.org
+    release_builds:
+      default: lunar/release-build.yaml
+      ds: lunar/release-stretch-build.yaml
+      dsv8: lunar/release-stretch-arm64-build.yaml
+      uxhf: lunar/release-armhf-build.yaml
+      uxv8: lunar/release-arm64-build.yaml
```
Comment (dirk-thomas) [inline index.yaml:56]: This file doesn't exist.

**Pull request merged**